.github/workflows/update-resources.yml

```diff
@@ -0,0 +1,28 @@
+name: Update Resources
+on:
+  schedule:
+    - cron: "0 4 * * *"
+
+jobs:
+  update-resources:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-java@v4
+        with:
+          distribution: temurin
+          java-version: 11
+
+      - name: Setup Gradle
+        uses: gradle/actions/setup-gradle@v3
+
+      - name: Execute Resource Generator
+        run: ./gradlew generateResources
+
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v6
+        with:
+          title: "chore: update rarity data from wiki"
```

Comment:
can we use `${{ steps.date.outputs.date }}` here so the PRs are at least a little distinguishable

Reply:
would need the appropriate step added to make this output

Reply:
Done in https://github.com/pajlads/DinkPlugin/pull/452/commits/190d6e1e7d1bce55458c765ea53f4b27e59d8095